### PR TITLE
[zh] Docs for seperated tidb-dashboard deployment

### PR DESCRIPTION
**Pull request by SabaPing**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s): 
   - Code: https://github.com/pingcap/tidb-operator/pull/4787

### CODE REVIEW

**Review by lilyjazz — APPROVED** [review]

File: zh/access-dashboard.md [hunk 1]
```diff
@@ -36,60 +47,51 @@ spec:
...
+export SERVICE_NAME=${cluster_name}-discovery && \
+export PORT=10261 && \
+export HTTP_PATH=/dashboard
+```
+
+### 访问独立部署的 TiDB Dashboard
+
+你需要使用 v1.1.1 版本及以上的 TiDB Operator 以及 v4.0.1 版本及以上的 TiDB 集群。
+
+请先参照 [TiDB Operator 中不支持的 Dashboard 功能](get-started.md#部署独立的 TiDB Dashboard) 部署独立的 TiDB Dashboard。
```
Comment (shichun-0415) [inline zh/access-dashboard.md:64]:
```suggestion
访问前，确保你已经[部署独立的 TiDB Dashboard](get-started.md#部署独立的-tidb-dashboard)。
```

File: zh/deploy-on-alibaba-cloud.md [hunk 2]
```diff
@@ -287,6 +289,16 @@ mysql --comments -h ${tidb_lb_ip} -P 4000 -u root
 >
 > 出于安全考虑，假如你已经或将要配置 VPN 用于访问 VPC，强烈建议将 `db-monitor.yaml` 文件里 `spec.grafana.service.annotations` 中的 `service.beta.kubernetes.io/alicloud-loadbalancer-address-type` 设置为 `intranet` 以禁止监控服务的公网访问。
 
+## 访问 TiDB Dashboard Web UI
+
+你可以通过浏览器访问 `<tidb-dashboard-exposed>:12333` 地址查看 Grafana 监控指标。
+
+`tidb-dashboard-exposed` 是集群 TiDB Dashboard Service 的 LoadBalancer IP。
```
Comment (shichun-0415) [inline zh/deploy-on-alibaba-cloud.md:296]:
```suggestion
`tidb-dashboard-exposed` 是集群 TiDB Dashboard Service 的 `LoadBalancer` IP。
```

**Review by mornyx — APPROVED** [review]

**Comment by mornyx** [comment]
LGTM

File: zh/deploy-on-gcp-gke.md [hunk 3]
```diff
@@ -166,8 +166,9 @@ kubectl create namespace tidb-cluster
 {{< copyable "shell-regular" >}}
 
 ```shell
-curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-cluster.yaml &&
-curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-monitor.yaml
+curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-cluster.yaml && \
+curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-monitor.yaml %% \
```
Comment (qqqdan) [inline zh/deploy-on-gcp-gke.md:170]:
```suggestion
curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-monitor.yaml && \
```

File: zh/access-dashboard.md [hunk 4]
```diff
@@ -18,11 +18,19 @@ TiDB Dashboard 是从 TiDB 4.0 版本起引入的可视化面板，用于帮助
...
 
-## 前置条件
+## 前置条件：确定 TiDB Dashboard 的 service
+
+本章节介绍在不同部署方式下，如何确定 TiDB Dashboard 的 service 和 HTTP 路径，后续访问 TiDB Dashboard 只需要将这一小节的确定的 service 和 HTTP 路径填入对应配置文件即可。
+
+目前 TiDB Dashboard 在集群中有两种部署方式，两种方式都可以访问 TiDB Dashboard，你可以根据需要选择其一：
+
+- 内嵌在 PD 进程中。这种部署方式下 TiDB Dashboard 位于 PD web server 的 `/dashboard` 路径中，其他路径可能无法访问。该部署方式会在后续 TiDB release 中去除，因此建议使用独立部署的 TiDB Dashboard。
+- 作为独立的服务。这种部署方式下 TiDB Dashboard 是独立的 StatefulSet，并且有专用的 service。Web server 的路径可以通过 `TidbDashboard.spec.pathPrefix` 配置。
```
Comment (hfxsd) [inline zh/access-dashboard.md:30]:
```suggestion
- 作为独立的服务。这种部署方式下 TiDB Dashboard 是独立的 StatefulSet，并且有专用的 service。Web server 的路径可以通过 `TidbDashboard.spec.pathPrefix` 配置。
- 内嵌在 PD 进程中。这种部署方式下 TiDB Dashboard 位于 PD web server 的 `/dashboard` 路径中，其他路径可能无法访问。注意该部署方式会在后续 TiDB release 中去除，因此建议使用独立部署的 TiDB Dashboard。
```

File: zh/access-dashboard.md [hunk 5]
```diff
@@ -36,60 +44,53 @@ spec:
...
+export HTTP_PATH=/dashboard
+```
+
+### 访问独立部署的 TiDB Dashboard
+
+你需要使用 v1.1.1 版本及以上的 TiDB Operator 以及 v4.0.1 版本及以上的 TiDB 集群。
+
+访问前，确保你已经[部署独立的 TiDB Dashboard](get-started.md#部署独立的-tidb-dashboard)。
+
+这种方法部署的 TiDB Dashboard，service、port 和 HTTP 路径如下(默认值)：
```
Comment (hfxsd) [inline zh/access-dashboard.md:63]:
```suggestion
这种方法部署的 TiDB Dashboard，service、port 和 HTTP 路径如下（默认值）：
```

File: zh/get-started.md [hunk 6]
```diff
@@ -348,6 +348,31 @@ tidbcluster.pingcap.com/basic created
...
+如果访问 Docker Hub 网速较慢，可以使用 UCloud 上的镜像：
+
+{{< copyable "shell-regular" >}}
+
+```
+kubectl -n tidb-cluster apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/basic-cn/tidb-dashboard.yaml
+```
+
+<details>
+<summary>点击查看期望输出</summary>
```
Comment (hfxsd) [inline zh/get-started.md:368]: 只有一行输出结果，似乎不需要用 details 样式。可以直接改成正文。

Comment (shichun-0415) [reply zh/get-started.md:368]: 前面的章节都是这种风格，我感觉可以保留，让风格一致。

**Review by hfxsd — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**